lib/fastlane/actions/mailgun.rb

```diff
@@ -0,0 +1,108 @@
+module Fastlane
+  module Actions
+    class MailgunAction < Action
+
+      def self.is_supported?(platform)
+        true
+      end
+
+      def self.run(options)
+        require 'rest_client'
+
+        handle_exceptions(options)
+
+        mailgunit(options[:mailgun_apikey],
+                  options[:mailgun_sandbox_domain],
+                  options[:mailgun_sandbox_postmaster],
+                  options[:to],
+                  options[:subject],
+                  compose_mail(options[:message],options[:success]))
+
+      end
+
+      def self.description
+        "Send a success/error message to an email group"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
+                                       env_name: "MAILGUN_SANDBOX_DOMAIN",
+                                       description: "Mailgun sandbox domain for your mail"),
+          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_postmaster,
+                                       env_name: "MAILGUN_SANDBOX_POSTMASTER",
+                                       description: "Mailgun sandbox domain postmaster for your mail"),
+          FastlaneCore::ConfigItem.new(key: :mailgun_apikey,
+                                       env_name: "MAILGUN_APIKEY",
+                                       description: "Mailgun apikey for your mail"),
+          FastlaneCore::ConfigItem.new(key: :to,
+                                       env_name: "MAILGUN_TO",
+                                       description: "Destination of your mail"),
+          FastlaneCore::ConfigItem.new(key: :message,
+                                       env_name: "MAILGUN_MESSAGE",
+                                       description: "Text of your mail"),
+          FastlaneCore::ConfigItem.new(key: :subject,
+                                       env_name: "MAILGUN_SUBJECT",
+                                       description: "Subject of your mail",
+                                       optional: true,
+                                       is_string: true,
+                                       default_value: "Fastlane Build"),
+          FastlaneCore::ConfigItem.new(key: :success,
+                                      env_name: "MAILGUN_SUCCESS",
+                                      description: "Was this build successful? (true/false)",
+                                      optional: true,
+                                      default_value: true,
+                                      is_string: false)
+
+        ]
+      end
+
+      def self.author
+        "thiagolioy"
+      end
+
+      private
+      def self.compose_mail(text,success)
+        text << "\n Git Author: #{Actions.git_author}"
+        text << "\n Last Commit: #{Actions.last_git_commit}"
+        text << "\n Success: #{success}"
+      end
+
+      def self.handle_exceptions(options)
+          unless ENV['MAILGUN_APIKEY']
```

Comment:
You have to access those values using `options[:api_key]`, etc. otherwise this won't work. In general, you'll never have to directly access `ENV`.